housekeeper/cli/tables.py

```diff
@@ -87,3 +89,67 @@ def get_versions_table(rows: List[dict]) -> Table:
             version_obj["expires_at"].split("T")[0] if version_obj["expires_at"] else "",
         )
     return table
+
+
+def squash_names(list_of_files):
+    """If subsequent elements (filenames) in 'list_of_files' end in an integer- And that integer is
+    following the previous- those are squashed when displayed.
+    Example:
+
+        ["asdf1.txt", "asdf2.txt"] becomes asdf[1-2].txt'
+    """
+    list_of_squashed = []
+    tag_list = []
+    if list_of_files == []:
+        return list_of_squashed
+    head, *tail = list_of_files
```

Comment:
What is happening here?

Reply:
Extended iterable unpacking, https://www.python.org/dev/peps/pep-3132/

head will be bound to first element of the list, tail will be _the rest_.